scripts/report_event.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+###############################################################################
+# vim: tabstop=4:shiftwidth=4:expandtab:
```

Comment:
Please move this comment black to be within the HEREDOC below, the triple quotes.

Reply:
Not sure what you mean. Are you saying you want lines 12-15 moved back up to start at line 4? If so, ok.

Reply:
Actually not ok. This is what you have in both the examples in the SDK (edited for width):
#!/usr/bin/env python
#######################################################################
# vim: tabstop=4:shiftwidth=4:expandtab:
# Copyright (c) 2017 SIOS Technology Corp. All rights reserved.
#######################################################################
"""
<Relevant comment>
"""

All I'm doing is adding more comment lines after the Copyright notice. What's the problem with that?

Reply:
Thanks for clarifying. Fixed.